iina/PlayerCore.swift

```diff
@@ -668,11 +668,18 @@ class PlayerCore: NSObject {
   }
 
   func frameStep(backwards: Bool) {
+    // When playback is paused the display link is stopped in order to avoid wasting energy on
+    // It must be running when stepping to avoid slowdowns caused by mpv waiting for IINA to call
+    // mpv_render_report_swap.
+    mainWindow.videoView.displayActive()
     if backwards {
       mpv.command(.frameBackStep)
     } else {
       mpv.command(.frameStep)
     }
+    if info.isPaused {
```

Comment:
After you send `.frameStep` or `.frameBackStep` to mpv, the playback will always stop, regardless the playback status before the command.